source/dsm.js

```diff
@@ -100,10 +100,25 @@ const dsm = ({
 
   const reducer = createReducer(defaultState, reducerMap);
   const actionCreators = actionNames.reduce((acs, description) => {
-    acs[description] = payload => ({
-      type: actions[description],
-      payload
-    });
+    acs[description] = (payload, meta) => {
+      const actionObj = {
+        type: actions[description]
+      };
+
+      if (payload) {
```

Comment:
What if the payload is `false`?  Also, what's the value of filtering payload (using `if`)?